int/tests/conftest.py

```diff
@@ -168,3 +169,55 @@ async def http_endpoint(agent: BaseAgent):
     with suppress(asyncio.CancelledError):
         await server_task
     await agent.cleanup()
+
+
+@pytest.fixture
+async def make_did():
+    """DID factory fixture"""
+
+    def _make_did():
+        did = create_did.asyncio(client=backchannel)
+        return did
+
+    yield _make_did()
+    # TODO create DID deletion method
+
+
+@pytest.fixture
+async def make_endorser_did(make_did):
+    """Endorser DID factory fixture"""
+
+    def _make_endorser_did():
+        did = make_did()
+        print("Publishing DID through https://selfserve.indiciotech.io")
+        response = httpx.post(
+            url="https://selfserve.indiciotech.io/nym",
+            json={
+                "network": "testnet",
+                "did": did.result.did,
+                "verkey": did.result.verkey,
+            },
+        )
+        if response.is_error:
+            print("Failed to publish DID:", response.text)
+            return
+        print("DID Published")
+        return did
+
+    yield _make_endorser_did
+
+
+@pytest.fixture
+async def accept_taa(scope="session"):
+    result = describe(
+        "Retrieve Transaction Author Agreement from the ledger", fetch_taa
+    )(client=issuer).result
+
+    result = describe("Sign transaction author agreement", accept_taa)(
+        client=issuer,
+        json_body=TAAAccept(
+            mechanism="on_file",
+            text=result.taa_record.text,
+            version=result.taa_record.version,
+        ),
+    )
```

Comment:
`describe` was a tool specific to the `acapy-revocation-demo` so this will raise an error, being unable to find the `describe` method. This method would basically just wrap the execution with some prints. The equivalent (without the wrapping prints) would be something like:

```suggestion
    result = await fetch_taa.asyncio(client=issuer).result
    result = await accept_taa.asyncio(
        client=issuer,
        json_body=TAAAccept(
            mechanism="on_file",
            text=result.taa_record.text,
            version=result.taa_record.version,
        ),
    )
```

Reply:
We'll also need to make sure the fetch_taa and accept_taa apis are imported.